README.md

```diff
@@ -36,3 +42,49 @@ SonarJS rules for ESLint
 [`no-use-of-empty-return-value`]: ./docs/rules/no-use-of-empty-return-value.md
 [`prefer-immediate-return`]: ./docs/rules/prefer-immediate-return.md
 [`prefer-single-boolean-return`]: ./docs/rules/prefer-single-boolean-return.md
+
+## Prerequisites
+
+Node.js (>=6.x).
+
+## Usage
+
+* If you don't have ESLint yet configured for your project follow [these instructions](https://github.com/eslint/eslint#installation-and-usage).
+* Install `eslint-plugin-sonarjs` using `npm` (or `yarn`) for you project or globally:
+
+```sh
+npm install eslint-plugin-sonarjs --save-dev # install for your project
+npm install eslint-plugin-sonarjs -g         # or install globally
+```
+
+* Add `eslint-plugin-sonarjs` to the `plugins` option of your `.eslintrc`:
+
+```json
+{
+  "plugins": ["sonarjs"]
+}
+```
+
+* Add `plugin:sonarjs/recommended` to the `extends` option to enable all rules:
```

Comment:
@stas-vilchik "**all** rules": it's true for now, but as we discussed some rules might be disabled in recommended profile